templates/sachet.yaml.erb

```diff
@@ -0,0 +1,7 @@
+<% require 'yaml' -%>
```

Comment:
hey, could you convert the template to epp? that's the new templating standard and we would like to have all new templates in epp and not erb format.

Reply:
I don't know much (it's my first work on a module) but yes, I'll have a look :)

Reply:
Done in https://github.com/voxpupuli/puppet-prometheus/pull/590/commits/44b919241836bfa8762aa98de831f530ee071695